articles/architecture-scenarios/_includes/_authorization/_introduction.md

```diff
@@ -38,3 +41,7 @@ For [API level integration](#api-integration), Auth0 supports both [custom claim
 ::: panel Best Practice
 When deciding whether you should use permissions through custom claims or scopes, you should make sure you understand the nature and purpose of scopes.  There is a nice [blog post](https://auth0.com/blog/on-the-nature-of-oauth2-scopes/) on that which is easy to read and helps clear up the topic.
 :::
+
+<% if (platform === "b2b") { %>
+For many multi-organization applications, it is important to know which organization an access token applies to. Taking care to follow the [best practices](#Organization-Data-in-an-Access-Token) can save you time and effort.
```

Comment:
Also referring here to id token, in a fashion similar to the comment above (https://github.com/auth0/docs/pull/8546/files#r356446439)

```suggestion
For multi-organization scenarios, it can often important to know which organization an access token (or even and id token) applies to. Taking care to follow the [best practices](#Organization-Data-in-an-Access-Token) can save you time and effort.
```
The location associated with the has fragment `#Organization-Data-in-an-Access-Token` doesn't seem to be a  valid one on this page.

Reply:
```suggestion
For multi-organization scenarios, it can often be important to know which organization an access token (or even an ID token) applies to. Taking care to follow the [best practices](#organization-data-in-an-access-token) can save you time and effort. 
```